developer-docs-site/docs/sdks/ts-sdk/sdk-transport-layer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Typescript SDK Transport Layer"
+slug: "typescript-sdk-transport-layer"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The transport layer in the SDK provides a robust and reliable communication channel between the client-side application and the blockchain server. It exports a [Provider](./sdk-transport-layer.md) class that extends both the Aptos REST API and the Aptos Indexer.
+The transport layer acts as a mediator between the client-side application and the blockchain server, ensuring reliable and efficient communication.
+
+These classes provide a high-level interface for the application to interact with the blockchain server. The classes are designed to be easy to use and understand, allowing developers to quickly integrate the SDK into their applications.
+
+### Provider class
```

Comment:
Let's use `##` here instead and `###` below? I figure this is left over from when it was in the big doc.